app/views/datafiles/_date_field.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group" id="<%= period %>_date_form_group">
+  <fieldset>
+    <legend>
+      <span class="form-label-bold">
+          <%= period.humanize %> Date
+          <span class="form-hint">For example, 31 3 2016</span>
+      </span>
+    </legend>
+
+    <div class="form-date">
+      <div class="form-group form-group-day">
+        <label for="#<%= period %>_day">Day</label>
+        <%= text_field_tag "datafile[#{period}_day]", nil, id: "#{period}_day", class: "form-control" %>
```

Comment:
Just asking the question if it's ok that this is `nil`, i.e. that when you're on the edit page everything is auto-populated other than the date...